Resources/Prototypes/Entities/Clothing/Neck/medals.yml

```diff
@@ -19,4 +19,70 @@
   - type: Sprite
     sprite: Clothing/Neck/Medals/gold.rsi
   - type: Clothing
-    sprite: Clothing/Neck/Medals/gold.rsi  
\ No newline at end of file
+    sprite: Clothing/Neck/Medals/gold.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckCargomedal
+  name: cargo medal
+  description: Given for the best work in the cargo department.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Medals/cargomedal.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Medals/cargomedal.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckEngineermedal
+  name: engineer medal
+  description: Given for the best work in the engineering department.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Medals/engineermedal.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Medals/engineermedal.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckMedicalmedal
+  name: medical medal
+  description: Given for the best work in the medical department.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Medals/medicalmedal.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Medals/medicalmedal.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckSciencemedal
+  name: science medal
+  description: Given for the best work in the science department.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Medals/sciencemedal.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Medals/sciencemedal.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckSecuritymedal
+  name: security medal
+  description: Given for the best work in the security department.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Medals/securitymedal.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Medals/securitymedal.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckClownmedal
+  name: clown medal
+  description: Given for the best joke in the whole space. HONK!

```

Comment:
"the whole space" sounds wrong

Reply:
whats black and white and red all over? the mime and he died to blunt head trauma.